.github/workflows/build-and-deploy-production.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         context: ./
         platforms: linux/amd64
         tags: |
-          ghcr.io/sidestream-tech/unified-auctions-ui/frontend:${{ github.sha }}
+          ghcr.io/sidestream-tech/auctions-ui-prod/frontend:${{ github.sha }}
```

Comment:
Isn't the issue is not with the namespace, but with the name of the image? Can't it just be:
- `ghcr.io/sidestream-tech/unified-auctions-ui/frontend-prod` here and
- `ghcr.io/sidestream-tech/unified-auctions-ui/frontend-staging` in the other file?

Reply:
Both solutions work fine for now we can either separate by registry or separate by image name. Pick one that looks nicer for you :)